common/taskana-common/src/main/resources/sql/postgres/taskana_schema_update_4.3.0_to_4.8.0_postgres.sql

```diff
@@ -5,3 +5,43 @@ SET search_path = %schemaName%;
 INSERT INTO TASKANA_SCHEMA_VERSION (VERSION, CREATED) VALUES ('4.8.0', CURRENT_TIMESTAMP);
 
 ALTER TABLE TASK ADD COLUMN RECEIVED TIMESTAMP NULL;
+ALTER TABLE TASK ADD COLUMN MANUAL_PRIORITY INT NULL;
+ALTER TABLE TASK ADD COLUMN CUSTOM_INT_1 INT NULL;
+ALTER TABLE TASK ADD COLUMN CUSTOM_INT_2 INT NULL;
+ALTER TABLE TASK ADD COLUMN CUSTOM_INT_3 INT NULL;
+ALTER TABLE TASK ADD COLUMN CUSTOM_INT_4 INT NULL;
+ALTER TABLE TASK ADD COLUMN CUSTOM_INT_5 INT NULL;
+ALTER TABLE TASK ADD COLUMN CUSTOM_INT_6 INT NULL;
+ALTER TABLE TASK ADD COLUMN CUSTOM_INT_7 INT NULL;
+ALTER TABLE TASK ADD COLUMN CUSTOM_INT_8 INT NULL;
+
+ALTER TABLE CONFIGURATION ADD COLUMN CUSTOM_ATTRIBUTES CLOB NULL;
```

Comment:
CLOB is unsupported in Postgres. Pleas use type text

Reply:
ok